libexec/exx-install

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env sh
+# Usage: exx install
+# Summary: Install `exx` globally
+
+PATH_INSTALL=$HOME/.sub
```

Comment:
Would it be possible to let the user define an INSTALL_PATH in a ```.subrc``` file that would contain something like

INSTALL_PATH=<path>

If such a ~/.subrc file exists, we could source it, otherwise, use the default $HOME/.sub